slack_cli_hooks/hooks/start.py

```diff
@@ -4,25 +4,12 @@
 import sys
 
 from slack_cli_hooks.error import CliError
-from slack_cli_hooks.hooks.utils import ManagedOSEnvVars
 from slack_cli_hooks.protocol import Protocol, build_protocol
 
 PROTOCOL: Protocol
 
 DEFAULT_MAIN_FILE = "app.py"
 
-SLACK_CLI_XOXB = "SLACK_CLI_XOXB"
-SLACK_CLI_XAPP = "SLACK_CLI_XAPP"
-SLACK_BOT_TOKEN = "SLACK_BOT_TOKEN"
-SLACK_APP_TOKEN = "SLACK_APP_TOKEN"
-
-
-def validate_env() -> None:
-    if not os.environ.get(SLACK_CLI_XOXB):
-        raise CliError(f"Missing local run bot token ({SLACK_CLI_XOXB}).")
-    if not os.environ.get(SLACK_CLI_XAPP):
-        raise CliError(f"Missing local run app token ({SLACK_CLI_XAPP}).")
```

Comment:
No harm in removing this IMO since an error will eventually be caught in the app if these variables are expected! I imagine this guard was to ensure hooks are called from the CLI but I'm honestly not sure 🤔

Reply:
If I remember correctly this was additional safeguard that was originally put in but we don't need it anymore